sponnet/pipeline.libsonnet

```diff
@@ -309,6 +309,7 @@
       withManifests(manifests):: self + if std.type(manifests) == 'array' then { manifests: manifests } else { manifests: [manifests] },
       withMoniker(moniker):: self + { moniker: moniker },
       withSkipExpressionEvaluation():: self + { skipExpressionEvaluation: true },
+      withOverrideNamespace(namespace):: self + { namespaceOverride: namespace },
```

Comment:
@Mahito are you open to changing the func name to match the key name as @kskewes suggested?

Reply:
@louisjimenez @kskewes Thank you for your comment and I'm sorry for the late changing. I  fixed it.